src/main/java/org/isf/visits/gui/VisitView.java

```diff
@@ -317,11 +375,36 @@ private JButton getAddVisitSecondButton() {
 		}
 		return addSecondVisitButton;
 	}
+	
+	private JButton getDeleteSecondVisitButton() {
+		if (deleteSecondVisitButton == null) {
+			deleteSecondVisitButton = new JButton(MessageBundle.getMessage("angal.visit.removevisit.btn"));
+			deleteSecondVisitButton.setMaximumSize(new Dimension(VISIT_BUTTON_WIDTH, ALL_BUTTON_HEIGHT));
+			deleteSecondVisitButton.setHorizontalAlignment(SwingConstants.LEFT);
+			deleteSecondVisitButton.addActionListener(actionEvent -> {
+
+				int row = jTableSecond.getSelectedRow();
+				if (row < 0) {
+					MessageDialog.info(VisitView.this, MessageBundle.getMessage("angal.common.pleaseselectarow.msg"));
+					return;
+				}
+				Visit visit = (Visit) jTableSecond.getModel().getValueAt(row, -1);
+				int ok = MessageDialog.okCancel(VisitView.this, MessageBundle.getMessage("angal.visit.removevisit.msg"));
```

Comment:
The okCancel() method does the message lookup for you; only pass the key and remove the MessageBundle.getMessage()

Reply:
Done, thanks!